garage/tf/misc/tensor_utils.py

```diff
@@ -123,7 +123,14 @@ def pad_tensor_dict(tensor_dict, max_len):
     return ret
 
 
-class enclosing_scope(object):
+def normalized_inputs(inputs, instance=()):
+    if isinstance(inputs, dict):
+        return type(instance)(inputs.values())
```

Comment:
values() is not guaranteed to have a stable order. you may need to sort by keys first.